notes/0.2.1.markdown

```diff
@@ -1,12 +1,18 @@
 
-### interface
+### breaking changes
 
-In sbt-datatype 0.2.1 the `protocol` type, which generates abstract class, is renamed to `interface`.
-It supports `methods` list as well as `fields`. [#14][14] by [@Duhemm][@Duhemm]/[#19][19] by [@eed3si9n][@eed3si9n]
+- The `protocol` type, which used to generate abstract classes, is renamed to `interface` [#19][19] by [@eed3si9n][@eed3si9n]
+- The `enumeration` type is renamed to `enum`, and the list of values are now called `symbols` (instead of `types`).
```

Comment:
You describe them as a list of values, so why not call them values? Is it also from Avro?